src/rtabmap_manager/launch/rtabmap_manager.launch

```diff
@@ -7,7 +7,6 @@
     <arg name="rgb_topic"               default="$(arg depth_cam_node)/rgb/image_rect_color" />
     <arg name="depth_topic"             default="$(arg depth_cam_node)/depth/depth_registered" />
     <arg name="camera_info_topic"       default="$(arg depth_cam_node)/rgb/camera_info" />
-    <arg name="depth_camera_info_topic" default="$(arg depth_cam_node)/depth/camera_info" />
```

Comment:
Why did you get rid of this?

Reply:
It was unused in the launchfile. Just checked the zed-ros-example that most of the launch stuff originated from, and it's not used there either.